src/servo.rs

```diff
@@ -385,7 +385,7 @@ fn test_route_wildcard_complex() {
 
 pub fn route_request(request: Request, configs: &Configuration) -> Response {
     let (args, callback) = configs.server.route_request(&request, &configs.routes);
-    callback(request.with_args(args), configs)
+    callback(request.with_url_args(args), configs)
```

Comment:
shouldn't this also have `request.with_url_args(args).with_query_params(params)`?